packages/commands/coverage/src/index.ts

```diff
@@ -163,30 +163,51 @@ function renderCoverage(coverage: SchemaCoverage) {
     }
   }
 
-  Logger.log(
-    `Types covered: ${
-      coverage.stats.numTypes > 0
-        ? ((coverage.stats.numTypesCovered / coverage.stats.numTypes) * 100).toFixed(1)
-        : 'N/A'
-    }%`,
-  );
-  Logger.log(
-    `Types covered fully: ${
-      coverage.stats.numTypes > 0
-        ? ((coverage.stats.numTypesCoveredFully / coverage.stats.numTypes) * 100).toFixed(1)
-        : 'N/A'
-    }%`,
-  );
-  Logger.log(
-    `Fields covered: ${
-      coverage.stats.numFields > 0
-        ? ((coverage.stats.numFiledsCovered / coverage.stats.numFields) * 100).toFixed(1)
-        : 'N/A'
-    }%`,
-  );
+  const logStatsResult = [
```

Comment:
maybe we should strongly type this variable? `{method: string, result: string}[]`?